src/components/interface/filter/FilterMenu.jsx

```diff
@@ -0,0 +1,96 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import Button from '@material-ui/core/Button';
+import FilterIcon from '@material-ui/icons/PlaylistAdd';
+import Menu from '@material-ui/core/Menu';
+import MenuItem from '@material-ui/core/MenuItem';
+
+import SubMenu from './SubMenu';
+
+class FilterMenu extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      anchorEl: null
+    };
+  }
+
+  handleMenuOpen = (event) => {
+    this.setState({ anchorEl: event.currentTarget });
+  };
+
+  handleMenuClose = () => {
+    this.setState({ anchorEl: null });
+  };
+
+  renderChoices = (name, opt) => {
+    const { filtered } = this.props;
```

Comment:
For these helper functions, can we add a new line between destructures and returns